notebooker/serialization/mongo.py

```diff
@@ -411,7 +423,12 @@ def get_count_and_latest_time_per_report(self, subfolder: Optional[str]):
         for report, all_runs in jobs_by_name.items():
             latest_start_time = max(r["job_start_time"] for r in all_runs)
             scheduled_runs = len([x for x in all_runs if x.get("scheduler_job_id")])
-            output[report] = {"count": len(all_runs), "latest_run": latest_start_time, "scheduler_runs": scheduled_runs}
+            output[report] = {
+                "count": len(all_runs),
+                "latest_run": latest_start_time,
+                "scheduler_runs": scheduled_runs,
+                "category": r["category"],
```

Comment:
This key will not exist if it is not in the mongo document.
```suggestion
                "category": r.get("category"),
```